tech-far-hub/src/pages/get-started/template-default.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import type { HeadFC, PageProps } from "gatsby";
+import { graphql } from "gatsby";
+import Layout from "../../components/layout";
+
+const LifecyclePage: React.FC<PageProps<Queries.GetStartedInnerPageQuery>> = ({
```

Comment:
I guess I should look at the mocks as to what this LifeCyclePage means. At first I thought it had to do with React's component LifeCycle but now I'm thinking it might be a page on the site that has to do with the LifeCycle of some tech thing ?

Reply:
Oh yeah it's a procurement lifecycle :D Like, the pages will document the various things you need to do to have a procurement.